include/openthread/platform/radio.h

```diff
@@ -621,6 +622,38 @@ bool otPlatRadioGetPromiscuous(otInstance *aInstance);
  */
 void otPlatRadioSetPromiscuous(otInstance *aInstance, bool aEnable);
 
+/**
+ * Sets the rx-on-when-idle state to the radio platform.
+ *
+ * Radio driver implementations supporting OT_RADIO_CAPS_RX_ON_WHEN_IDLE capability should consider idle periods the
+ * following situations:
```

Comment:
```suggestion
 * There are a few situations that the radio can enter sleep state if the 
 * device is in rx-off-when-idle state but it's hard and costly for the SubMac
 * to identify these situations and instruct the radio to enter sleep:
```
I think this may be better for understanding.